examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -183,6 +188,25 @@ class GalleryDrawer extends StatelessWidget {
       selected: Theme.of(context).platform == TargetPlatform.iOS,
     );
 
+    final List<Widget> textSizeItems = <Widget>[];
+    final Map<double, String> textSizes = <double, String>{
+      -1.0: 'System Default',
```

Comment:
I'd probably just use null as the sentinel

Reply:
Whoa!  Cool!  I didn't do it initially, because I assumed that `null` wasn't a valid Map key.  Switched to `null`.